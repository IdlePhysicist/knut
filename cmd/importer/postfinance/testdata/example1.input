Buchungsart:;Alle Buchungen
Konto:;CHXXXXXXXXXXXXXXXXXXX
Währung:;CHF
Buchungsdatum;Avisierungstext;Gutschrift in CHF;Lastschrift in CHF;Valuta;Saldo in CHF
2020-11-02;"Desc1";;-140.00;2020-11-03;
2020-11-05;"Desc2";;-35.55;2020-11-06;17566.87
2020-11-07;"Desc3";1.00;;2020-11-08;17902.42
2020-11-16;"Desc4";;-11.00;2020-11-17;

Disclaimer:
Dies ist kein durch PostFinance AG erstelltes Dokument. PostFinance AG ist nicht verantwortlich für den Inhalt.