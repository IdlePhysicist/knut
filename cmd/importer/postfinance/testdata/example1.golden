2020-11-02 "Desc1"
Assets:Postfinance      TBD                            140 CHF

2020-11-05 "Desc2"
Assets:Postfinance      TBD                          35.55 CHF

2020-11-07 "Desc3"
TBD                     Assets:Postfinance               1 CHF

2020-11-16 "Desc4"
Assets:Postfinance      TBD                             11 CHF

